src/app/debug/debug-tab.service.ts

```diff
@@ -0,0 +1,14 @@
+import { Injectable } from '@angular/core';
+import { Observable, Subject } from 'rxjs';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class DebugTabService {
+  private refreshSubject: Subject<void> = new Subject();
+  refresh$: Observable<void> = this.refreshSubject.asObservable();
+
+  refresh(): void {
+    this.refreshSubject.next();
+  }
+}
```

Comment:
Is this pr also part of the refresh issue? Because I see here a new refresh subject

Reply:
I created it but haven't implemented it yet, implementation will be done in my next project